docs/guide/counter/cmd/countercli/commands/counter.go

```diff
@@ -20,64 +21,59 @@ var CounterTxCmd = &cobra.Command{
 	Long: `Add a vote to the counter.
 
 You must pass --valid for it to count and the countfee will be added to the counter.`,
-	RunE: counterTxCmd,
+	RunE: doCounterTx,
```

Comment:
Personally a fan of making the RunE command an unexported version of the command which it is a part of - I guess this is stylistic, but I've done mostly all of them this way - aka here it would be `counterTxCmd` - nbd though